media-plugins/vdr-newsticker/vdr-newsticker-0.0.4-r1.ebuild

```diff
@@ -9,13 +9,12 @@ DESCRIPTION="VDR plugin: Show rdf Newsticker on TV"
 HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
 SRC_URI="http://vdr.websitec.de/download/${PN}/${P}.tar.gz"
 
-KEYWORDS="~amd64 ~x86"
-SLOT="0"
 LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
 
-BDEPEND="acct-user/vdr
-	media-video/vdr"
-DEPEND="${BDEPEND}"
+BDEPEND="acct-user/vdr"
```

Comment:
Doesn't need this also be in ```RDEPEND```?

Reply:
Hello @ConiKost ,
basically correct, media-video/vdr includes the dependency for acct-user/vdr - so it has an indirect dependency (see COMMON_DEPEND in media-video/vdr .

As far as I understood, BDEPEND="acct-user/vdr" is necessary on the build host when there are commands like "chown vdr:vdr" in the ebuild?!?